linux/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3'
+services:
+  mmdvmhost:
+    build: https://github.com/f4hlv/MMDVMHost.git
+#   image: f4hlv/mmdvmhost
```

Comment:
References you, probably best without reference to an individual.

Reply:
This is not a reference. There are several ways to get a docker image. Build it yourself or download from the docker hub.
The docker image f4hlv / mmdvmhost is built automatically as soon as there is a change on the G4KLX's master.
To update MMDVMHost just execute two commands:
- docker-compose pull
- docker-compose up -d

The other way is to recreate the image itself. The installation depends on the preferences of the user.

Reply:
My approach is to download an official docker image of G4KLX docker hub.
Replace https://hub.docker.com/r/f4hlv/mmdvmhost/ with https://hub.docker.com/r/g4klx/mmdvmhost/